addons/html_builder/static/src/sidebar/block_tab.scss

```
@@ -65,6 +65,20 @@ $o-we-zindex: $o-we-overlay-zindex + 1 !default;
         z-index: 0;
         position: relative;
 
+        @for $i from 2 through 4 {
+            &.o_snippets_container_body--col-#{$i} {
```

CSS classes names conventions are a bit part of our "oral tradition" and are really context/module dependent (yes, I know..). Let's just say to try to stay stick the the current file  ;)

Also, `.o_snippets_container_body--col-3` doesn't really clarify if this element is a column with "size 3" or a container with 3 columns 🤔 I'd use BS approach, see https://getbootstrap.com/docs/5.3/layout/grid/#row-columns.

```suggestion
            &.o_snippets_container_body_row_cols_#{$i} {
```